From d8c6efd2464782b1c313d0bb5c8e3dbde86a673d Mon Sep 17 00:00:00 2001
Message-Id: <d8c6efd2464782b1c313d0bb5c8e3dbde86a673d.1616414938.git.avarab@gmail.com>
In-Reply-To: <20210306192525.15197-1-avarab@gmail.com>
References: <20210306192525.15197-1-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 5 Mar 2021 14:36:16 +0100
Subject: [PATCH v4 1/4] Makefile/coccicheck: add comment heading for all
 SPATCH flags
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In 960154b9c17 (coccicheck: optionally batch spatch invocations,
2019-05-06) the comment being re-added here was amended and reworded
to apply to the SPATCH_BATCH_SIZE variable.

If we'd like a separate comment on SPATCH_FLAGS now the diff to
introduce it will need to also move the comment. So add this heading
back to make subsequent diffs smaller.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f3dc2178324..eef99b4705d 100644
--- a/Makefile
+++ b/Makefile
@@ -1198,10 +1198,11 @@ PTHREAD_CFLAGS =
 SPARSE_FLAGS ?=
 SP_EXTRA_FLAGS = -Wno-universal-initializer
 
+# For the 'coccicheck' target
+SPATCH_FLAGS = --all-includes --patch .
 # For the 'coccicheck' target; setting SPATCH_BATCH_SIZE higher will
 # usually result in less CPU usage at the cost of higher peak memory.
 # Setting it to 0 will feed all files in a single spatch invocation.
-SPATCH_FLAGS = --all-includes --patch .
 SPATCH_BATCH_SIZE = 1
 
 include config.mak.uname
-- 
2.31.0.285.gb40d23e604f

